### PR TITLE
Add uninstalled plugins and descriptions on "plugins" command

### LINKED ISSUE #597: Update `plugins` command and add plugin description
When users download neo-cli, they don't really know how many plug-ins we have and what they're for.

Update the `plugins` command as following:

```bash
> plugins
Installed plugins:
        LevelDBStore  Uses LevelDB to store the blockchain data
Uninstall plugins:
       RocksDBStore Uses RocksDBStore to store the blockchain data
       SystemLog   Prints the consensus log
       RpcServer     Enables RPC for the node
       ApplicationLogs  Synchronizes the smart contract log with the NativeContract log (Notify)
       RpcNep5Tracker  Enquiries NEP-5 balance and transactions history of accounts through RPC
       StatesDumper  Exports Neo-CLI status data
```

### PR DESCRIPTION
**Pull request by joeqian10**
Close #597

### CODE REVIEW

File: neo-cli/CLI/MainService.Plugins.cs [hunk 1]
```diff
@@ -96,14 +98,30 @@ private void OnUnInstallCommand(string pluginName)
...
         private void OnPluginsCommand()
         {
+            Dictionary<string, string> dic = new Dictionary<string, string>();
+            dic.Add("ApplicationLogs", "Synchronizes the smart contract log with the NativeContract log (Notify)");
+            dic.Add("LevelDBStore", "Uses LevelDB to store the blockchain data");
+            dic.Add("RocksDBStore", "Uses RocksDBStore to store the blockchain data");
+            dic.Add("RpcNep5Tracker", "Enquiries NEP-5 balance and transactions history of accounts through RPC");
+            dic.Add("RpcServer", "Enables RPC for the node");
+            dic.Add("StatesDumper", "Exports Neo-CLI status data");
+            dic.Add("SystemLog", "Prints the consensus log");
```
Comment (Tommo-L) [inline neo-cli/CLI/MainService.Plugins.cs:108]: Remove `SystemLog` plugin, as we'll remove it now.

File: neo-cli/CLI/MainService.Plugins.cs [hunk 2]
```diff
@@ -96,14 +98,30 @@ private void OnUnInstallCommand(string pluginName)
...
             {
-                Console.WriteLine("Loaded plugins:");
-                Plugin.Plugins.ForEach(p => Console.WriteLine("\t" + p.Name));
+                Console.WriteLine("Installed plugins:");
+                Plugin.Plugins.ForEach(p =>
+                    {
+                        var description = dic.GetValueOrDefault(p.Name, "Description");
+                        Console.WriteLine("\t" + p.Name + "\t" + description);
+                        dic.Remove(p.Name);
+                    });
```
Comment (Tommo-L) [inline neo-cli/CLI/MainService.Plugins.cs:118]: Insert one more blank line after installed plugin list.

File: neo-cli/CLI/MainService.Plugins.cs [hunk 3]
```diff
@@ -96,14 +98,30 @@ private void OnUnInstallCommand(string pluginName)
...
+                Console.WriteLine("Uninstalled plugins:");
+                dic.ForEach(p => Console.WriteLine("\t" + p.Key + "\t" + p.Value));
             }
             else
             {
-                Console.WriteLine("No loaded plugins");
+                Console.WriteLine("No installed plugins");
```
Comment (Tommo-L) [inline neo-cli/CLI/MainService.Plugins.cs:124]: `else` branch is incorrect. When `Plugin.Plugins.Count = 0`, it'll only print `No installed plugins` without uninstalled plugin list.

File: neo-cli/CLI/MainService.Plugins.cs (hunk 1 above)
Comment (joeqian10) [reply neo-cli/CLI/MainService.Plugins.cs]: Done.

File: neo-cli/CLI/MainService.Plugins.cs (hunk 2 above)
Comment (joeqian10) [reply neo-cli/CLI/MainService.Plugins.cs]: Done.

File: neo-cli/CLI/MainService.Plugins.cs (hunk 3 above)
Comment (joeqian10) [reply neo-cli/CLI/MainService.Plugins.cs]: I have removed this else branch since it's always > 0 because of the SystemLog plugin.

**Comment by erikzhang** [comment]
https://github.com/neo-project/neo-node/issues/597#issuecomment-637290080

**Comment by Tommo-L** [comment]
Waiting for https://github.com/neo-project/neo/pull/1678

**Comment by ProDog** [comment]
Tested with https://github.com/neo-project/neo-modules/pull/257, good👏
```
NEO-CLI v3.0.0-preview2  -  NEO v3.0.0-preview2  -  NEO-VM v3.0.0-preview2

neo> plugins
Loaded plugins:
        SystemLog
        ApplicationLogs Synchronizes the smart contract log with the NativeContract log (Notify)
        LevelDBStore    Uses LevelDB to store the blockchain data
        RocksDBStore    Uses RocksDBStore to store the blockchain data
        RpcNep5Tracker  Enquiries NEP-5 balance and transactions history of accounts through RPC
        RpcServer       Enables RPC for the node
        StatesDumper    Exports Neo-CLI status data
neo>
```

**Pull request merged**